luisy/targets.py

```diff
@@ -242,6 +250,64 @@ def read(self):
         return self.spark.table(self.table_uri)
 
 
+class AzureBlobStorageTarget(SparkTarget):
+    file_ending = "parquet"
+
+    def __init__(
+            self,
+            container=None,
+            account=None,
+            abfss_path=None,
+            inferschema=False,
+    ):
+        self.container = container
+        self.account = account
+        self.abfss_path = abfss_path
+        self.inferschema = inferschema
+
+    def make_dir(self, path):
+        pass
+
+    def remove(self):
+        from py4j.java_gateway import java_import
```

Comment:
Probably code that will break in future ans we definitely have to touch (due to all the private methods), may be a bit risky that we delete data accidentally. 

As we always overwrite, we could also leave this function void for a start?

Reply:
Decided to leave it out and to not remove for a start